frontend/src/static/js/v_summary.js

```diff
@@ -379,6 +379,12 @@ window.vSummary = {
       });
     },
 
+    getRandomColor() {
+      const maxHexColorValue = 16777213;
```

Comment:
Should this be 16777214 since the `Math.random` is not inclusive of the upper bound?

Reply:
Yes u are right